enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/errorhandling/PowerAuthActivationOtpException.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.onboardingserver.common.errorhandling;
+
+import io.getlime.security.powerauth.rest.api.spring.exception.PowerAuthActivationException;
+
+/**
+ * Exception thrown in case activation using OTP code fails (soft fail).
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+public class PowerAuthActivationOtpException extends PowerAuthActivationException {
+
+    private static final long serialVersionUID = -1587779018046975797L;
```

Comment:
Nitpick, I guess the value `1` is fine for us as well.

Reply:
Not consistent with the rest of the stack, I will not use it unless we change it in the rest of the stack.